recipe/meta.yaml

```
@@ -38,16 +50,18 @@ requirements:
 test:
   commands:
     - test -f $PREFIX/include/lemon/glpk.h                 # [unix]
-    - test -f $PREFIX/lib/libemon.a                        # [unix]
+    - test -f $PREFIX/lib/libemon.so                       # [linux]
+    - test -f $PREFIX/lib/libemon.dylib                    # [osx]
     - if not exist %LIBRARY_INC%\\lemon\\glpk.h exit 1     # [win]
     - if not exist %LIBRARY_LIB%\\lemon.lib exit 1         # [win]
 
 about:
   home: https://lemon.cs.elte.hu/trac/lemon
-  license: Boost-1.0
+  license: BSL-1.0
   license_file: LICENSE
   summary: 'Library for Efficient Modeling and Optimization in Networks'
 
 extra:
   recipe-maintainers:
     - bluescarni
+    - stuarteberg
```

```suggestion
    - stuarteberg
    - k-dominik
```
@stuarteberg, @k-dominik would you be willing to be added as maintainers?

---

Yes

---

sure, thank you for adding me